pyanaconda/anaconda.py

```diff
@@ -228,7 +228,10 @@ def initInterface(self):
         # from kernel boot command line like this
         if "webui" in kernel_arguments:
             from pyanaconda.ui.webui import CockpitUserInterface
-            self._intf = CockpitUserInterface(None, self.payload)
+            self._intf = CockpitUserInterface(
+                None, self.payload,
+                "webui.remote" in kernel_arguments
+            )
```

Comment:
It might be useful to have each parameter on a new line.

Reply:
Updated and repushed.